jinahub/encoders/image/TimmImageEncoder/timm_encoder.py

```diff
@@ -0,0 +1,102 @@
+from typing import Dict, Iterable, List, Optional, Tuple
+
+import numpy as np
+import torch
+import torchvision.transforms as T
+from jina import DocumentArray, Executor, requests
+from jina_commons.batching import get_docs_batch_generator
+from timm import create_model
+from timm.data import resolve_data_config
+from timm.data.transforms_factory import create_transform
+
+
+class TimmImageEncoder(Executor):
+    """
+    :class:`TimmImageEncoder` encodes ``Document`` blobs of type `ndarray`
+    (`float32`) and shape `H x W x C` into `ndarray` of `D`.
```

Comment:
I think you can just put `3` instead of `C`. Also, clarify that the image needs to be in RGB format